tachyon/crypto/commitments/kzg/shplonk.h

```diff
@@ -107,18 +107,6 @@ class SHPlonk : public UnivariatePolynomialCommitmentScheme<
               y, low_degree_extensions_vec[i]);
         });
 
-    for (size_t i = 0; i < low_degree_extensions_vec.size(); ++i) {
-      for (size_t j = 0; j < grouped_poly_openings_vec[i].points.size(); ++j) {
-        const Point& p = *grouped_poly_openings_vec[i].points[j];
-        for (size_t k = 0; k < low_degree_extensions_vec[i].size(); ++k) {
-          CHECK_EQ(
```

Comment:
Being expressed "redundant", is this check also being done somewhere?

Reply:
It checks whether the element of `low_degree_extensions_vec` are really linear combination of given `grouped_poly_openings_vec[i].poly_openings_vec[k]` and these functionality should be tested in unittests and actually we do!